idaes_examples/tests/test_browse.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+from idaes_examples import browse
+
+
+@pytest.mark.unit
```

Comment:
I am not sure about how the testing of the notebooks is done, but is this test module necessary?

Reply:
@lbianchi-lbl could you please help with this question.